common/app/experiments/Experiments.scala

```diff
@@ -90,6 +90,15 @@ object HeaderTopBarSearchCapi
       participationGroup = Perc1B,
     )
 
+object OphanEsm
```

Comment:
Not sure what happened here. Looks like this is coming from https://github.com/guardian/frontend/pull/26564